### DISCUSSION TITLE
Vmax

### DISCUSSION THREAD

**Discussion by Pooja-gupta15** [Open-Systems-Pharmacology/Forum#1011] [Questions & Problems]
how do I convert Vmax value given as pmol/min/mg of cytosolic protein to pmol/mL/min.

**Comment by prvmalik** [comment]
Hi,

Scaling of in vitro Vmax to in vivo is incredibly uncertain if not completely impossible.

With that said, you can still make a guess.

What type of experiment was your Vmax derived from: Hepatocytes? Liver microsomes? Recombinant enzyme systems?

**Reply by Pooja-gupta15** [reply]
Hello, Thank you for your kind reply.

The in vitro experiment was performed on liver cytosolic protein. Particularly Vmax value for Aldehyde oxidase is given but I need to convert its units so that I can use it as an input parameter in my model.
